### DISCUSSION TITLE
Mobius Sync IOS path query

### DISCUSSION THREAD

**Discussion by thorrrr** [MobiusSync/MobiusSync#90] [Q&A]
Hi Guys I know this is a commercial product which i have not purchased yet until i know it works. But it is built on Syncthing i am aware of Apple tying down like it does. But i am easily able to sync folders and files which go into the mobius app itself. But i have an app that will only work in 2 folders

iCloud Drive/Logseq/filename
On my iPhone/Logseq/filename
Question is there anyway to sync directly into On my iPhone/Logseq/ without having to go into the Mobius folder? If so please can you explain how to achieve this ?

**Comment by SimonPickup** [comment]
Not yet possible. Will need #59
